<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sConfigEQ</class>
 <widget class="QWidget" name="sConfigEQ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QTreeWidget" name="UI_TW_DEVICE_CEQ">
       <column>
        <property name="text">
         <string notr="true">设备树</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>232</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>设备组</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>设备组IP:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="UI_PB_LOCAL_ADDRESS">
               <property name="maximumSize">
                <size>
                 <width>64</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>本地地址</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>1</number>
             </property>
             <item>
              <widget class="QLineEdit" name="UI_LE_IP0">
               <property name="text">
                <string>192</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="UI_LE_IP1">
               <property name="text">
                <string>168</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="UI_LE_IP2">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="UI_LE_IP3">
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>设备组名称：</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLineEdit" name="UI_LE_DEVICE_NAME"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="UI_PB_ADD_DEVICE">
            <property name="text">
             <string>添加设备组</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="UI_PB_DELETE_DEVICE">
            <property name="text">
             <string>删除设备组</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QTableWidget" name="UI_TW_SVID1">
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>5</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>6</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>7</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>8</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>9</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>10</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>SVID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>名字</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="UI_TW_SVID2">
       <row>
        <property name="text">
         <string>11</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>12</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>13</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>14</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>15</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>16</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>17</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>18</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>19</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>20</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>SVID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>名字</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <row>
        <property name="text">
         <string>21</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>22</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>23</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>24</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>25</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>26</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>27</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>28</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>29</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>30</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>SVID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>名字</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_2">
       <row>
        <property name="text">
         <string>31</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>32</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>33</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>34</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>35</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>36</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>37</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>38</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>39</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>40</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>SVID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>名字</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="UI_PB_OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
